wrappers/csharp/Intel.RealSense/Context.cs

```diff
@@ -32,10 +32,11 @@ public Context()
         /// create a static snapshot of all connected devices at the time of the call
         /// </summary>
         /// <returns></returns>
-        public DeviceList QueryDevices()
+        public DeviceList QueryDevices(bool include_platform_camera = false)
         {
             object error;
-            var ptr = NativeMethods.rs2_query_devices(m_instance.Handle, out error);
+            var ptr = NativeMethods.rs2_query_devices_ex(m_instance.Handle,
+                include_platform_camera ? 0xff : 0xfe, out error);
```

Comment:
Can you replicate the enum in c# for maintainability?

Reply:
I don't want to complicate C# API with bit-fiend enums. If the need for this will arise from C# users, we can always add. For now, I'm only aware of the requirement to include / exclude platform-cameras)